discovery/chan_series.go

```diff
@@ -1,6 +1,7 @@
 package discovery
 
 import (
+	"fmt"
```

Comment:
the commit message doesn't have the required format

Reply:
oops - oversight. my bad